model/src/main/proto/screens.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+enum ScreenName {
```

Comment:
This & its constants should be documented.

Reply:
done.